languages/elixir/exercises/concept/pattern-matching/.docs/introduction.md

```diff
@@ -0,0 +1,195 @@
+
```

Comment:
Before doing a thorough review of this document, maybe you could first make it more concise? I would suggest to [read up on the introduction.md description](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd) to understand the goals and requirements for this file.

A good example of an existing Elixir exercise with a nice introduction document is the [booleans exercise](https://github.com/exercism/v3/blob/master/languages/elixir/exercises/concept/booleans/.docs/introduction.md).